apps/server/src/modules/task/repo/submission.repo.ts

```diff
@@ -35,6 +36,22 @@ export class SubmissionRepo extends BaseRepo<Submission> {
 		return result;
 	}
 
+	public async deleteUserFromTeams(userId: EntityId): Promise<number> {
```

Comment:
are you actually meaning group submissions ? because I think "Teams" is then a bit missleading here

Reply:
true, the old function call in service was submission.removeUserFromTeamMembers

Reply:
I also strogle by the teams naming. But it look like it is named teamMembers in submission as well. How it is called in the ui?